index.html

```diff
@@ -119,17 +119,19 @@ <h2 class="section-title slideanim">Bądź na bieżąco!</h2>
                 <div class="newsletter-box-img">
                     <img class="newsletter-img" src="img/man-in-space.jpg" alt="człowiek stojący na księżycu z flagą"/>
                 </div>
+
                 <div class="newsletter-box-form">
-                    <form action="http://hireux.pl/form.php" method="post" target="_blank">
+                    <div>
```

Comment:
ja bym zostawił to w <form></form>. Jeśli wewnątrz jest input i przycisk sumbit to jest to wg semantyki chyba formularz html i będzie nam potrzebny do późniejszego przetworzenia tych danych.